src/components/Header/NetworkSelector.tsx

```diff
@@ -146,40 +146,42 @@ const SelectorWrapper = styled.div`
 const StyledChevronDown = styled(ChevronDown)`
   width: 16px;
 `
-const BridgeLabel = ({ chainId }: { chainId: SupportedChainId }) => {
+
+function getBridgeLabel(chainId: SupportedChainId) {
```

Comment:
or maybe we wanna make it translate the `Bridge` text but keep the chain untranslated?

Reply:
If we don't translate bridge we still need to use translations for ordering of the words.